src/main/java/com/songrLab/songr/SongrApplication.java

```diff
@@ -24,9 +30,33 @@ public static void main(String[] args) {
 
 	@Controller
 	public class HelloController {
-		
+
+		@Autowired
+		private SongRepository songRepository;
 		@Autowired
-		AlbumRepo albumRepo;
+		private AlbumRepository albumRepository;
+
+
+		@PostMapping("/album")
```

Comment:
You've already defined an addAlbum method below. This one works, but is missing the step that saves the new album into the album repository.